src/style.css

```diff
@@ -1,3 +1,100 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0;
+  margin: 0 auto;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  --blue: #00acdc;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  margin-right: 50px;
+  text-align: center;
+  padding: 0 auto;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  height: 60px;
+  line-height: 60px;
+  display: inline-block;
+  margin: 0 auto;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.nav__link:hover {
+  color: var(--blue);
+}
+
+.is-active {
+  color: var(--blue);
+}
+
+.is-active::after {
+  position: absolute;
+  bottom: 0;
+  content: "";
+  display: block;
+  width: 37px;
```

Comment:
```suggestion
  width: 100%;
```

Reply:
it should work fine with all links

Reply:
![image](https://github.com/mate-academy/layout_moyo-header/assets/140541650/18452aa3-b6d2-4692-bf90-8e3661b9a729)

Reply:
now it covering the whole header with it

Reply:
ok a fixed it nvm